installer/pkg/config-generator/tls.go

```diff
@@ -121,10 +134,104 @@ func (c *ConfigGenerator) GenerateTLSConfig(clusterDir string) error {
 		DNSNames:     []string{baseAddress, fmt.Sprintf("%s.%s", "*", baseAddress)},
 		Subject:      pkix.Name{CommonName: baseAddress, Organization: []string{"ingress"}},
 		Validity:     validityThreeYears,
-		IsCA:         false}
+		IsCA:         false,
+	}
 
 	if _, _, err := generateCert(clusterDir, kubeCAKey, kubeCACert, ingressKeyPath, ingressCertPath, cfg); err != nil {
-		return fmt.Errorf("failed to generate ingress CAs: %v", err)
+		return fmt.Errorf("failed to generate ingress CA: %v", err)
+	}
+
+	// Kube admin certs
+	cfg = &tls.CertCfg{
+		KeyUsages:    x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature,
+		ExtKeyUsages: []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth, x509.ExtKeyUsageClientAuth},
+		Subject:      pkix.Name{CommonName: "system:admin", Organization: []string{"system:masters"}},
+		Validity:     validityThreeYears,
+		IsCA:         false,
+	}
+
+	if _, _, err = generateCert(clusterDir, kubeCAKey, kubeCACert, adminKeyPath, adminCertPath, cfg); err != nil {
+		return fmt.Errorf("failed to generate kube admin certificate: %v", err)
+	}
+
+	// Kube API server certs
+	apiServerAddress, err := cidrhost(c.Cluster.Networking.ServiceCIDR, 1)
+	if err != nil {
+		return fmt.Errorf("can't resolve api server host address: %v", err)
+	}
+	cfg = &tls.CertCfg{
+		KeyUsages:    x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature,
+		ExtKeyUsages: []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth, x509.ExtKeyUsageClientAuth},
+		Subject:      pkix.Name{CommonName: "kube-apiserver", Organization: []string{"kube-master"}},
+		DNSNames:     []string{fmt.Sprintf("%s-api.%s", c.Name, c.BaseDomain), "kubernetes", "kubernetes.default", "kubernetes.default.svc", "kubernetes.default.svc.cluster.local"},
```

Comment:
hint: We should align formats with the DNSNames field below so either always inline or always multiline, also those could be moved into constants